Sources/Common/Core/Math/index.js

```diff
@@ -17,14 +17,6 @@ function notImplemented(method) {
   return () => vtkErrorMacro(`vtkMath::${method} - NOT IMPLEMENTED`);
 }
 
-function vtkSwapVectors3(v1, v2) {
```

Comment:
you could keep it with the following signature: `vtkSwapMatrixColumns(matrix, column1, column2)` (or vtkSwapMatrixRows?)